core/deployment/pom.xml

```diff
@@ -79,6 +79,10 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
As @geoand mentioned, I don't think it's a good idea to add this library to core

Reply:
Hi @gastaldi ,
 if you check the code the reflection framework is used in more than one place. I've just put it on more centralized place. Anyway - what I've tried to achieve here is to register for reflection all classes in given package. Could you please advice is there an other way to achieve it?

Not a problem to move it only on Tika related deploynment, but as I said this library is used on more than one place. In my experience if such things is happened, we have to move the dependency on more centralized place, or to not used it at all and to replace it with the similar functionality.